db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190709163639) do
+ActiveRecord::Schema.define(version: 20190709163513) do
```

Comment:
We should probably figure out why merging master brings this change in but for now we can just remove it